src/recordlinker/routes/patient_router.py

```diff
@@ -65,6 +66,73 @@ def update_person(
         patient_reference_id=patient.reference_id, person_reference_id=person.reference_id
     )
 
+
+@router.post(
+    "/",
+    summary="Create a patient record and link to an existing person",
+    status_code=fastapi.status.HTTP_201_CREATED,
+)
+def create_patient(
+    payload: typing.Annotated[schemas.PatientCreatePayload, fastapi.Body],
+    session: orm.Session = fastapi.Depends(get_session),
+) -> schemas.PatientRef:
+    """
+    Create a new patient record in the MPI
+    """
+    person = service.get_person_by_reference_id(session, payload.person_reference_id)
+
+    if person is None:
+        raise fastapi.HTTPException(status_code=fastapi.status.HTTP_422_UNPROCESSABLE_ENTITY)
```

Comment:
```suggestion
        raise fastapi.HTTPException(status_code=fastapi.status.HTTP_422_UNPROCESSABLE_ENTITY, detail="Person not found. Must include a person_reference_id that already exists in the MPI.")
```

@ericbuckley How do you feel about more detailed error messages? I initially got a 422 when testing out locally because I hadn't merged in the changes from last week (the EO ones) and was confused about which part of the request was incorrect.